src/views/LoginView.vue

```diff
@@ -268,69 +260,80 @@ export default {
       this.showSnackBar = false;
       this.snackBarMessage = '';
     },
-   isAlphaNumeric(val){
-      return (/^[\w.@-]+$/).test(val) || 'Username must contain only letters or numbers';
+    isAlphaNumeric(val) {
+      return /^[\w.@-]+$/.test(val) || 'Username must contain only letters or numbers';
     },
-    has8OrMoreCharacters(val){
+    has8OrMoreCharacters(val) {
       return val.length >= 8 || 'Password must contain at least eight characters';
     },
+    scrollAndFocusLogin() {
+      this.$refs.loginContainer.scrollIntoView();
+      this.$refs.usernameInput.focus();
+    },
+    directToSignup() {
+      this.$router.push({ name: ROUTE_NAME_SIGNUP });
+      this.scrollAndFocusLogin();
+    }
+  },
+ mounted() {
+    if (this.isLoggingIn) {
+      this.scrollAndFocusLogin();
+    }
```

Comment:
How is the UX of this auto scroll on page mount? Can you share a video?

Reply:
Posted in the PR description